tests/lib/fakestore/store/store.go

```diff
@@ -177,7 +178,7 @@ type essentialInfo struct {
 var errInfo = errors.New("cannot get info")
 
 func snapEssentialInfo(w http.ResponseWriter, fn, snapID string, bs asserts.Backstore) (*essentialInfo, error) {
-	snapFile, err := snap.Open(fn)
+	file, err := snapfile.Open(fn)
```

Comment:
any particular reason to rename snapFile to file here?

Reply:
not really just that it made the format on save for Go code auto-add imports from the current project in VS code work otherwise it was confused with snapfile and snapFile, but I can revert this rename if you like

Reply:
I would revert it,  or s/file/f/ at most